kernel-crawler/rhel-subscription-crawler/crawler.py

```diff
@@ -226,9 +232,10 @@ def filter_repos(self, content_sets):
         return repos
 
 
-    def get_packages(self, repos):
+    def get_packages(self, repos, session):
         urls = set()
         for repo in repos:
+            logger.info(f'Processing repo {repo}')
             logger.debug(f'################ {repo} ################')
```

Comment:
Maybe we can remove the debug log, it seems a little redundant with the new info one.